NimbleMedium/Sources/Presentation/Modules/ArticleDetail/ArticleDetailView+UIModel.swift

```diff
@@ -16,13 +16,15 @@ extension ArticleDetailView {
         let articleUpdatedAt: String
         let authorName: String
         let authorImage: URL?
+        var authorFollowing: Bool
```

Comment:
small naming suggestion
```suggestion
        var authorIsFollowing: Bool
```